flintrock/config.yaml.template

```diff
@@ -1,6 +1,7 @@
 services:
   spark:
     version: 1.6.0
+    # preferred-mirror: # optional; default to 'https://s3.amazonaws.com/spark-related-packages/${file}'
```

Comment:
Ok, so to be quite clear, i will use these variables in my default value in the script : 
spark_version
spark_distribution